### PR TITLE
Update Python components

### PR DESCRIPTION
**Pull request by Perevalov**
The Helsinki NLP component was updated according to the new parameter naming and also the docker-compose file was created.

The REL-Falcon component is a new one.

### CODE REVIEW

File: qanary-component-MT-Python-Helsinki-NLP/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  mt-helsinki-nlp:
+    # image: qanary/qanary-component-mt-python-helsinki-nlp:0.1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    network_mode: host
+    # ports:
+      # - 40120:40120
+    restart: unless-stopped
+    environment:
```
Comment (anbo-de) [inline qanary-component-MT-Python-Helsinki-NLP/docker-compose.yml:12]: @Perevalov I think it is not useful to define in a public GitHub-hosted file internal configuration.

File: qanary-component-MT-Python-Helsinki-NLP/README.md [hunk 2]
```diff
@@ -54,8 +54,8 @@ docker run -d -p 41062:41062 \
     -e SPRING_BOOT_ADMIN_URL='https://webengineering.ins.hs-anhalt.de:43740' \
     -e SPRING_BOOT_ADMIN_USERNAME='admin' \
     -e SPRING_BOOT_ADMIN_PASSWORD='admin' \
-    -e SERVICE_HOST='http://webengineering.ins.hs-anhalt.de' \
-    -e SERVICE_PORT=41062 \
+    -e SERVER_HOST='http://webengineering.ins.hs-anhalt.de' \
+    -e SERVER_PORT=41062 \
```
Comment (anbo-de) [inline qanary-component-MT-Python-Helsinki-NLP/README.md:58]: @Perevalov I suggest updating it to some existing demo server or replace the concrete values with a placeholder.

File: qanary-component-MT-Python-Helsinki-NLP/boot.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo The port number is: $SERVER_PORT
+echo The Qanary pipeline URL is: $SPRING_BOOT_ADMIN_URL
+if [ -n $SERVER_PORT ]
+then
+    exec gunicorn --timeout 180 -b :$SERVER_PORT --access-logfile - --error-logfile - run:app
```
Comment (anbo-de) [inline qanary-component-MT-Python-Helsinki-NLP/boot.sh:7]: @Perevalov timeout could also be an optional variable. If it does not exist, you can use 180 as a default/fallback.

**Comment by Perevalov** [comment]
Related to #170

File: qanary-component-MT-Python-Helsinki-NLP/boot.sh (hunk 3 above)
Comment (Perevalov) [reply qanary-component-MT-Python-Helsinki-NLP/boot.sh]: Changed in https://github.com/WDAqua/Qanary-question-answering-components/pull/143/commits/a7c0c9970c1681511eb67aa716321edee8f0364d

File: qanary-component-MT-Python-Helsinki-NLP/docker-compose.yml (hunk 1 above)
Comment (Perevalov) [reply qanary-component-MT-Python-Helsinki-NLP/docker-compose.yml]: @anbo-de Changed in: https://github.com/WDAqua/Qanary-question-answering-components/pull/143/commits/a7c0c9970c1681511eb67aa716321edee8f0364d

**Comment by anbo-de** [comment]
@Perevalov There are still conflicts in this pull request.

**Comment by Perevalov** [comment]
> @Perevalov There are still conflicts in this pull request.

@anbo-de The conflicts were resolved.

**Comment by anbo-de** [comment]
@Perevalov Thanks! :+1:

**Pull request merged**